utils.py

```diff
@@ -721,6 +722,86 @@ def get_hashable_value(value):
         return value
 
 
+def convert_to_text_angular(html_data):
```

Comment:
Does this handle all the "problematic" cases you got from the run on the test server?

Also, we need to do a run on the real server to get the full set of problematic cases. Once #4969 has gone in please send an email to @AllanYangZhou and me, and I can help you run the script on the server. If this function works on all the cases we get from there, then you should be fine.

Reply:
Yes, it does handle. Okay sure, I'll mail.